helion/_compiler/inductor_lowering.py

```diff
@@ -676,6 +744,49 @@ def codegen_baddbmm(ctx: GraphInterpreter, node: torch.fx.Node) -> ast.AST:
     )
 
 
+def var_mean_helper_(
+    x: torch._inductor.ir.TensorBox,
+    *,
+    axis: list[int] | None,
+    correction: float | None,
+    keepdim: bool,
+    return_mean: bool,
+) -> torch._inductor.ir.IRNode:
+    from torch._inductor.lowering import to_dtype
+    from torch._inductor.lowering import var_mean_sum_
+    from torch._prims_common import get_computation_dtype
+
+    out_dtype = x.get_dtype()
+    compute_dtype = get_computation_dtype(out_dtype)
+    x = to_dtype(x, compute_dtype, copy=False)
+    kwargs = {
+        "x": x,
+        "axis": axis,
+        "correction": correction,
+        "keepdim": keepdim,
+        "return_mean": return_mean,
+    }
+    # TODO(yf225): support Welford reduction in Helion, then switch back to use Inductor `var_mean_helper_()`.
+    output = var_mean_sum_(**kwargs)
+    output = tuple(to_dtype(x, out_dtype, copy=False) for x in output)
+    return output[0] if not return_mean else output
+
+
+@register_inductor_lowering(  # pyre-ignore[56]
+    torch.ops.aten.var_mean.correction, lowering_dict=inductor_lowering_dispatch
```

Comment:
Won't this break compatbility with `torch==2.7`?  Until 2.8 is released we should support 2.7.

Reply:
sg - added 2.7 compat handling